app/service/rest.py

```diff
@@ -980,7 +1003,30 @@ def create_smtp_relay(service_id):
             status_code=500)
 
 
-def check_request_args(request):
+def check_unique_name_request_args(request):
+    name = request.args.get('name', None)
+    errors = []
+    if not name:
+        errors.append({'name': ["Can't be empty"]})
+    if errors:
+        raise InvalidRequest(errors, status_code=400)
+    return name
```

Comment:
```suggestion
    if not name:
        raise InvalidRequest({'name': ["Can't be empty"]}, status_code=400)
    return name
```

If you want to simplify only

Reply:
I believed in you and you broke my tests!

Always double check 'fast & risky' @AntoineAugusti is my new motto.

Reply:
Sorry about that!